tests/test_layers.py

```diff
@@ -2580,3 +2580,18 @@ def test_reverseV2(runner):
     data = np.random.rand(1, 2, 4).astype(np.float32)
     expected = model.predict(data)
     assert runner('tf_rev_v2', onnx_model, data, expected)
+
+
+@pytest.mark.skipif((is_tensorflow_older_than('1.14.0') or (not is_tf_keras)), reason='old tf version')
+def test_tensor_scatter_update(runner):
```

Comment:
Add
`@pytest.mark.skipif(get_maximum_opset_supported() < 11,
                    reason="TensorScatterUpdate is not supported before opset 11.")`

Reply:
Added. Thank you for the comment.